### PR TITLE
GCW- 2361- Move app version and item create from home screen

### PR DESCRIPTION
**Pull request by abulsayyad123**
Hi Team,
This is PR for JIRA ticket GCW-2361-Move app version and item create from home screen.  This PR includes:
* `Add inventory Item` is moved from dashboard to Menu and Item page.
* App version is moved from dashboard to Menu top (and it is available only for mobile).
* Changed and fixed test cases accordingly.
* Fixed test cases error related to mocking of `/summary` and `/booking_type` `GET` resource.
Please review the PR and let me know if any change is required.
Thanks.

### CODE REVIEW

File: app/controllers/search_code.js [hunk 1]
```diff
@@ -106,12 +107,15 @@ export default Ember.Controller.extend({
     },
 
     cancelSearch() {
+      let previousPath = this.get('previousPath');
       Ember.$("#searchText").blur();
       this.send("clearSearch", true);
       if(this.get("backToNewItem")) {
         this.transitionToRoute("items.new");
+      } else if (previousPath) {
+        this.transitionToRoute(previousPath);
```
Comment (Darshan-Chauhan) [inline app/controllers/search_code.js]: Can we only keep history.back()?
As it is more reliable and will also take to previous path and also less code in route to get the name of previous path

**Comment by Darshan-Chauhan** [comment]
@abulsayyad123 , just for future reference, you can add mockjax mocks to a variable and clear them after each test case is run.
So you won't get any issues of single test cases passing and same test cases failing in group. :)

File: app/controllers/search_code.js (hunk 1 above)
Comment (abulsayyad123) [reply app/controllers/search_code.js]: I wasn't very sure about `history.back()`, that is why I wrote `previousPath` logic. Just to be on safer side I kept `history.back()` as a last option. If `history.back()` is more reliable than we can go for it. 👍

Comment (Darshan-Chauhan) [reply app/controllers/search_code.js]: Yes, I think it would better :)

File: app/controllers/search_code.js [hunk 2]
```diff
@@ -111,7 +111,7 @@ export default Ember.Controller.extend({
       if(this.get("backToNewItem")) {
         this.transitionToRoute("items.new");
       } else {
-        this.transitionToRoute("index");
+        window.history.back();
```
Comment (namrataukirde) [inline app/controllers/search_code.js]: :+1:

**Pull request merged**